src/scenes/game-scene.ts

```diff
@@ -134,15 +138,27 @@ class GameScene extends Phaser.Scene {
     toggleMusic(this); // attaches listener to mute button
     const [isMute] = useLocalStorage('isMute', false);
     this.game.sound.mute = isMute; // set game mute to saved ls value
+
+    this.player = this.level.spawners.player.getChildren()[0] as Ben3;
+
+    const canvas = document.querySelector('#game');
+    canvas?.addEventListener('click', this.jump.bind(this));
```

Comment:
Does click fire with touch on mobile as well? I'm not able to test over the network still :(

Reply:
yeah you're right... it **doesn't work**

Reply:
done